spacy_transformers/layers/transformer_model.py

```diff
@@ -224,14 +224,11 @@ def _convert_transformer_inputs(model, wps: WordpieceBatch, is_train):
 
     hf_device = model.shims[0]._hfmodel.transformer.device
     kwargs = {
-        # Note: remove conversion to long when PyTorch >= 1.8.0.
```

Comment:
But there's not a corresponding change to the torch requirements?

Reply:
Fixed.